chpl/chpl-service/src/main/java/gov/healthit/chpl/scheduler/job/CertifiedProductDownloadableResourceCreatorJob.java

```diff
@@ -51,32 +56,43 @@ public CertifiedProductDownloadableResourceCreatorJob() throws Exception {
     @Override
     public void execute(JobExecutionContext jobContext) throws JobExecutionException {
         SpringBeanAutowiringSupport.processInjectionBasedOnCurrentContext(this);
+        LOGGER.info("********* Starting the Certified Product Downloadable Resource Creator job for {}. *********", edition);
 
         edition = jobContext.getMergedJobDataMap().getString("edition");
-        LOGGER.info("********* Starting the Certified Product Downloadable Resource Creator job for {}. *********", edition);
```

Comment:
Doesn't this info line need to be after the line that sets the edition? It was moved to a place where I think it'll always be `""`